EIPS/eip-4377.md

```diff
@@ -0,0 +1,360 @@
+---
+eip: 4377
```

Comment:
```suggestion
eip: 
```
ERC numbers are issued by editors & associates.  They are now sequential.

Once you have created a PR in the ERC repo a number will be manually issued.